### ISSUE TITLE
[FEATURE] add to playlist context menu

### ISSUE THREAD

**Issue by andreicociuba** [Tzahi12345/YoutubeDL-Material#404]
would it be awfully complicated to click on the  hamburger menu of the video and have an "add to playlist" option that would open another list to chose the playlist to add to?

it would be awesome!

also, thanks very much for all the work, i really do appreciate the responsiveness, work, and general attitude on this repository!

**Comment by GlassedSilver** [comment]
This reminds me that adding too many recommendations in one ticket can haunt the trackability of progress, because this idea had already been submitted by yours truly, the spammer #1 of issues (like 70% of all statistics on the internet, this one is completely out of thin air and unverified. :P)

https://github.com/Tzahi12345/YoutubeDL-Material/issues/214

Cheers

PS: evidently I'm very much FOR this idea.
PPS: yes, the attitude and chill vibes are definitely a big up for this repo!

**Comment by Tzahi12345** [comment]
Oops, didn't realize I missed that item from #214, actually there's a few there that maybe need to be broken into new issues as they're not implemented (dedicated playlist page and all).

Anyways I realized it's probably a bit tedious to use the create or edit playlist menu to add files to the playlist if you have thousands of videos, so that needs a refactor as well. Shouldn't be too hard as it's just a UI change.

Long story short, I added this ability in the [last couple commits](https://github.com/Tzahi12345/YoutubeDL-Material/commit/92413bd360397048402735a4509edd72aa2a7717)! Should be a good solution to the above problem until we get that resolved. Here's a screenshot, the `nightly` should be updated in an hour or so:

![image](https://user-images.githubusercontent.com/707413/127090001-d9476f8e-2666-417c-b912-ffdd94c2ad17.png)

If the file is already in a playlist, the add button will be disabled. Thanks for the suggestion, feel free to make more if you think of anything!
